codersdk/templates.go

```diff
@@ -133,3 +133,44 @@ func (c *Client) TemplateVersionByName(ctx context.Context, template uuid.UUID,
 	var templateVersion TemplateVersion
 	return templateVersion, json.NewDecoder(res.Body).Decode(&templateVersion)
 }
+
+type DAUEntry struct {
+	Date time.Time `json:"date"`
+	DAUs int       `json:"daus"`
```

Comment:
I think this should be `Amount`. It's the _amount_ of _DAUs_ on a specific _date_.